### DISCUSSION TITLE
Cannot make a selection that ends at the beginning of a line.

### DISCUSSION THREAD

**Discussion by lixiaoyan** [ghostty-org/ghostty#3992] [Feature Requests, Ideas] [answered]
https://github.com/user-attachments/assets/85a63c2e-b9a1-4644-bdab-1579119c29e3

Typically, I use this method to select multiple lines. However, it does not work with Ghostty. Ghostty will force the first character to be selected.

**Comment by rhodes-b** [comment, marked as answer]
there are duplicates of this (this is older)
https://github.com/ghostty-org/ghostty/discussions/5058

I know there is at least another I can't find it at the moment though

I will mark this as a duplicate thought since that as more interaction
